### DISCUSSION TITLE
0.5.3 Release: Architecture Overhaul - Revolt

### DISCUSSION THREAD

**Discussion by insertish** [stoatchat/discussions#369] [Announcements]
Read more about the new update here 👉 https://revolt.chat/posts/architecture-overhaul

**Comment by insertish** [comment]
Hello! We are trial running comments on our new blog, we haven't really decided whether we will be keeping them on or not, but for now here it is.
(reactions will be here to stay though!)

As always, remain civil.

**Comment by sussycatgirl** [comment]
:trollface:

**Reply by insertish** [reply]
:trollface:

**Reply by nyakowint** [reply]
oh you're such a :trollface:

**Reply by delta2571** [reply]
sus

**Reply by NoLogicAlan** [reply]
:trollface:

**Reply by Lutitious** [reply]
:trollface:

**Reply by Rexogamer** [reply]
Sus :trollface:

**Reply by ninjasmosa** [reply]
:trollface:

**Reply by infi** [reply]
:trollface:

**Reply by aeongdesu** [reply]
:trollface:

**Reply by FanMclaine** [reply]
:trollface:

**Reply by Ikun886-cxk** [reply]
:trollface:

**Reply by Rexogamer** [reply]
:trollface:

**Reply by fanticwastaken** [reply]
![trollface](https://user-images.githubusercontent.com/70902926/167724926-39ff3005-dd96-4e8b-b80d-48a10439fc0b.png)

**Reply by ghost** [reply]
:trollface:

**Comment by Inderix** [comment]
All of this looks great! Can't wait to see what you guys cook up next!

**Comment by ERROR-404-NULL-NOT-FOUND** [comment]
Finally, the fabled permissions update!

**Comment by MrChileball** [comment]
Nice!

**Comment by subterfugue** [comment]
Awesome! I'm excited to see what's coming from you guys!

**Comment by UnixPhonez** [comment]
End to End Encryption in dm's is a must also encryption for servers is a must!

**Reply by Rexogamer** [reply]
We're not currently planning to add E2EE to servers - aside from anything, a lot of servers have so many members that it would be somewhat pointless (a chain is only as strong as it's weakest link; someone could just download/leak the contents) and pretty difficult to implement/handle. DMs and groups, meanwhile, consist of a singular channel and are usually smaller; E2EE for them is still planned, although we can't promise any timescales as it's a bigger feature that will require a lot of work/is lower on our priority list.

**Comment by UnixPhonez** [comment]
Post a testflight link when the mobile version drops!
